--- a/SPECS/kernel.spec	2023-09-25 13:12:57.429570273 -0400
+++ b/SPECS/kernel.spec	2023-09-25 13:12:52.997543884 -0400
@@ -202,17 +202,17 @@
 %define with_headers   %{?_without_headers:   0} %{?!_without_headers:   1}
 %define with_cross_headers   %{?_without_cross_headers:   0} %{?!_without_cross_headers:   1}
 # perf
-%define with_perf      %{?_without_perf:      0} %{?!_without_perf:      1}
+%define with_perf      %{?_without_perf:      0} %{?!_without_perf:      0}
 # tools
-%define with_tools     %{?_without_tools:     0} %{?!_without_tools:     1}
+%define with_tools     %{?_without_tools:     0} %{?!_without_tools:     0}
 # bpf tool
-%define with_bpftool   %{?_without_bpftool:   0} %{?!_without_bpftool:   1}
+%define with_bpftool   %{?_without_bpftool:   0} %{?!_without_bpftool:   0}
 # kernel-debuginfo
-%define with_debuginfo %{?_without_debuginfo: 0} %{?!_without_debuginfo: 1}
+%define with_debuginfo %{?_without_debuginfo: 0} %{?!_without_debuginfo: 0}
 # kernel-abi-stablelists
 %define with_kernel_abi_stablelists %{?_without_kernel_abi_stablelists: 0} %{?!_without_kernel_abi_stablelists: 1}
 # internal samples and selftests
-%define with_selftests %{?_without_selftests: 0} %{?!_without_selftests: 1}
+%define with_selftests %{?_without_selftests: 0} %{?!_without_selftests: 0}
 #
 # Additional options for user-friendly one-off kernel building:
 #
